test/data_bundle_test.dart

```diff
@@ -23,7 +26,18 @@ void main() async {
       await dataItem.sign(wallet);
 
       expect(await dataItem.verify(), isTrue);
+    });
 
+    test('comfirm data item with wrong signaure fails verify', () async {
```

Comment:
nit spelling: `comfirm` to `confirm`

Reply:
[Fixed](https://github.com/ardriveapp/arweave-dart/pull/10/commits/d9b05af369fd17d80febafe91ad7c79deea47ddd)